Extensions/PrimitiveDrawing/shapepainterruntimeobject.js

```diff
@@ -60,6 +60,10 @@ gdjs.ShapePainterRuntimeObject.prototype.drawLine = function(x1, y1, x2, y2, thi
     this._renderer.drawLine(x1, y1, x2, y2, thickness);
 };
 
+gdjs.ShapePainterRuntimeObject.prototype.drawEllipse = function(x1, y1, width, height) {
```

Comment:
as x1 and y1 are the only points, and are the center coordinates, let's name them "centerX", "centerY".